volumina/__main__.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import contextlib
+import signal
+import sys
+
+import numpy
+import xarray
+from PyQt5.QtWidgets import QApplication
+
+from volumina import __version__
+from volumina.api import Viewer
+from volumina.colortables import default16_new
+from volumina.pixelpipeline.datasources import ArraySinkSource, ArraySource
+from volumina.layer import ColortableLayer, GrayscaleLayer
+
+
+@contextlib.contextmanager
+def volumina_viewer():
+    app = QApplication(sys.argv)
+    v = Viewer()
+    v.editor.setInteractionMode("navigation")
+    yield v
+    try:
+        signal.signal(signal.SIGINT, lambda *_: app.quit())
+        app.exec_()
+    finally:
+        app.quit()
+
+
+def axiorder_type(value):
+    reasons = []
+    if len(set(value)) != len(value):
+        reasons.append("Each axis may only appear ones")
+
+    if any(dim not in "txyzc" for dim in value):
+        reasons.append("Unrecognized axis value encountered. Only allowed axes: 'txyzc'.")
+
+    if reasons:
+        reasons.append(f"Got '{value}'.")
+        raise argparse.ArgumentTypeError(" ".join(reasons))
+
+    return value
+
+
+def parse_args():
+    p = argparse.ArgumentParser(
+        description="",
+        usage="",
+        epilog="",
+    )
+    p.add_argument("image", help="Path to .npy image")
+    p.add_argument("axistags", help="Strings describing axes in image. Valid values: 'tzyxc'", type=axiorder_type)
```

Comment:
Maybe this argument should be an option? Some other type of `image` could have the axis info in the future.

Reply:
yeah, let's decide about this in the future then (of course you are correct, I already have a branch to read ilasitk h5 output that already has axistags :))

Reply:
I was suggesting the following:
```suggestion
    p.add_argument("--axistags", help="Strings describing axes in image. Valid values: 'tzyxc'", type=axiorder_type, required=True)
```

Later, we'll drop `required=True` and decide on an image type basis.